packages/editor/src/services/ComponentShelfCategoriesState.ts

```diff
@@ -134,6 +134,7 @@ export const ComponentShelfCategoriesState = defineState({
         ScreenshareTargetComponent,
         LookAtComponent
       ]
+      // Ecommerce: [ECommerceComponent]
```

Comment:
let's remove this comment, the ecommerce will be injected from the ecommerce project